### PR TITLE
feat(vite-plugin-angular): add support for .ag extension

### PR DESCRIPTION
**Pull request by joshuamorony**
## PR Checklist

Allow `.ag` extension as a shorthand alternative to `.analog`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts [hunk 1]
```diff
@@ -368,7 +368,9 @@ export function angular(options?: PluginOptions): Plugin[] {
           }
 
           if (
-            (id.endsWith('.analog') || id.endsWith('.agx')) &&
+            (id.endsWith('.analog') ||
+              id.endsWith('.agx') ||
```
Comment (ajitzero) [inline packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts:372]: question: What exactly is `*.agx`? `agx` isn't set up in all the places where `ag` is set up now. Looks like an `mdx` equivalent with a markdown focus?

Comment (joshuamorony) [reply packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts:372]: Essentially yes, it allows using analog components as content files with front matter

Comment (ajitzero) [reply packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts:372]: Ah, okay. I was under the impression that they were all just aliases of each other. Thanks!

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**